launcher/ui/dialogs/skins/SkinManageDialog.cpp

```diff
@@ -139,6 +139,9 @@ void SkinManageDialog::on_fileBtn_clicked()
 {
     auto filter = QMimeDatabase().mimeTypeForName("image/png").filterString();
     QString raw_path = QFileDialog::getOpenFileName(this, tr("Select Skin Texture"), QString(), filter);
+    if (raw_path.isEmpty()) {
```

Comment:
```suggestion
    if (raw_path.isNull()) {
```
should be sufficient